.github/workflows/build-jobs.yaml

```diff
@@ -299,3 +300,14 @@ jobs:
           do
             ydb-ya-make  ${udfs_prefix}/${udf_name}
           done
+
+          systempython_udfs_prefix="../ytsaurus/ydb/ydb/library/yql/udfs/common/python/system_python"
```

Comment:
You need to put them in qt image somewhere in qt Dockerfile

Reply:
I actually don't need, just like udfs above this line

[QT build script copies](https://github.com/ytsaurus/ytsaurus/blob/main/yt/docker/ytsaurus/build.sh#L127) all `lib*.so` files from build directory into `yql`, and Dockerfile [copies the whole](https://github.com/ytsaurus/ytsaurus/blob/main/yt/docker/ytsaurus/Dockerfile#L127) `yql` folder